SETS/Shrinking.txt

```diff
@@ -0,0 +1,247 @@
+/**** RESTART ****/
```

Comment:
I think it would be a good idea to have a minimalistic setup file here. Copy paste already minimalistic ShearTemplate.txt and change variables of interest

Reply:
This way it would be easier to compare and highlight the compressibility feature. Also if we would add a test in a future compressible vs non-compressible, that would be cool

Reply:
I see. We have tried to work from ShearTemplate but it did not work. It was not general enough since the radius of the inclusion has to be tied to one specific user parameter. Also Shrinking setup includes additional feature than ShearTemplate (see PR with passive markers).